CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ set(Slicer_ORGANIZATION_DOMAIN "kitware.com")
 set(Slicer_ORGANIZATION_NAME   "Kitware, Inc.")
 
 # Default home and favorite modules
-set(Slicer_DEFAULT_HOME_MODULE "Home")
+set(Slicer_DEFAULT_HOME_MODULE "VPAWVisualize")
```

Comment:
I guess this is supposed to configure the application such that the VPAWVisualize module is selected upon launch. But that isn't the case when I launch. IDK why

Reply:
This was a side effect of not using the 3D Slicer Extension Wizard.  It is now resolved.